### PR TITLE
Just forked your repo for GIS homework!

### PR DESCRIPTION
**Pull request by yuxinnie**
Hi Diana. Your code is excellent!
I tried to run all the chunks in Rmd file but it seemed there was a little problem when I was trying to read raw data. Then I noticed the folder 'wk4prat' was missing. So I just add it. You can check it in the commit history:)
After that, I ran all the chunks smoothly! Also, I learned from you about the 'countrycode' package, brilliant.

(ps. I'm the second person above you on the google spreadsheet as the person below me did not have an Rmd file with gis code.
So please don't get too surprised if you receive pull requests twice. Thx)

### CODE REVIEW

**Comment by DianaMathai** [comment]
Hi,

Thank you! 

The here package was working fine when I re-run my existing code, to check. Hence I was confused on the suggested path change. To humbly brief you on the issue: Among many of my attempts to push the files prior, to couple of repositories on GIT, specifically to figure out the issues related to my push pull attempts from the Rstudio. I did not register that I created a file with the name wk4prat, the name did not replicate on my desktop folder where the Git was initiated, hence I could not trace it back, when I run your code. I figured out the issue that you would have faced while pulling my gis_code repo, and your suggestion as well. 
To summarize on the way forward. 
I have now placed all the files within the folder where git was initiated i.e. gis_code, for a seamless access, of all the files. Hence I shall not be accepting your pull request.  
To my understanding it would be clearer if I delete the 'wk4prat' to avoid confusion. But since u have already made the changes and run a successful code, I feel it would be a waste of effort if I delete the 'wk4prat'. 
Hence, I have decided on letting the folder stay populated with replicated data.

**Comment by DianaMathai** [comment]
My Apologies for the inconvenience. 

In-addition, Thank you, for letting me know of the issue. I learn something new today!! I hope to do better next time!!
